src/ripple/ledger/impl/View.cpp

```diff
@@ -1355,7 +1355,7 @@ rippleSend (ApplyView& view,
     {
         // VFALCO Why do we need this bCheckIssuer?
         // Direct send: redeeming IOUs and/or sending own IOUs.
-        terResult   = rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
+        rippleCredit (view, uSenderID, uReceiverID, saAmount, false, j);
```

Comment:
We do not: notice what happens two lines down: `terResult = tesSUCCESS`. There's a reason for this (even if it's a bit esoteric): the only errors that `rippleCredit` can return come from `trustCreate` and, obviously, there's no trust line that needs to be created if the sender or the receiver are the issuer (it must exist) or if XRP is being sent.